master/reference/public-cloud/aws.md

```diff
@@ -20,13 +20,13 @@ on your deployment.
 Calico requires the following security group exceptions to function properly
 in AWS.
 
-| Description      | Type            | Protocol | Port Range |
-|:-----------------|:----------------|:---------|:-----------|
-| BGP              | Custom TCP Rule | TCP      | 179        |
-| \*IPIP           | Custom Protocol | IPIP     | all        |
+| Description   | Type            | Protocol | Port Range |
+|:--------------|:----------------|:---------|:-----------|
+| BGP           | Custom TCP Rule | TCP      | 179        |
+| IPIP          | Custom Protocol | IPIP     | all        |
 
->\*IPIP: This rule is required only when using Calico with IPIP encapsulation.
-Keep reading for information on when IPIP is required in AWS.
+\* The IPIP exception is required only when using Calico with IPIP encapsulation. Keep reading 
```

Comment:
What does the `\*` do here?  It doesn't look like it is needed as part of the content; is it a formatting thing?

Reply:
It is a formatting thing, exactly. The `\` escapes the `*`, which would otherwise get rendered as a bullet. So, this is the way to do table footnotes where you have a `*` in the table and then below, you also need a `*` that represents the table note/footnote.

Reply:
However, I'm glad you mentioned this because I was missing the asterisk in the table, have added this.